master/docs/manual/cfg-statustargets.rst

```diff
@@ -1639,13 +1641,19 @@ Specifically, it follows the `Updating build status for commits <https://develop
 
 It uses the standard Python Twisted Agent to make REST requests to the stash server.
 It uses HTTP Basic AUTH.
-As a result, we recommend you use https in your base_url rather than http.
+As a result, we recommend you either connect over a secured network
```

Comment:
Please do not wrap the line here, we follow [one sentence per line](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) policy.